### DISCUSSION TITLE
can you send me the image of software how it looks like ?

### DISCUSSION THREAD

**Discussion by BeingTraveler** [GaganHonor/Face-Identification-System-#5] [Q&A] [answered]
please tell

**Comment by GaganHonor** [comment, marked as answer]
![Screenshot 2023-08-03 161407](https://github.com/GaganHonor/Face-Identification-System-/assets/141204441/c16c5ba3-dd99-436f-8d31-6af21483407d)

already on readme 😒but thanks for asking question i am here 24/7😊

**Reply by BeingTraveler** [reply]
thanks
